lib/Makefile

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2020 Matheus Rodrigues <matheussr61@gmail.com>
```

Comment:
why are you copying this makefile if you have a submodule?

Reply:
This is the makefile of lunatik user space module. What are you suggesting is to build the lunatik user space module from the submodule makefile?